src/store/reducers/appReducer.js

```diff
@@ -104,14 +111,16 @@ const deviceGroupListSchema = new schema.Array(deviceGroupSchema);
 const initialState = {
   ...errorPendingInitialState,
   deviceGroups: {},
+  deviceGroupFilters: {},
   activeDeviceGroupId: undefined,
   theme: 'dark',
   version: undefined,
   releaseNotesUrl: undefined,
   logo: svgs.contoso,
-  name: 'companyName',
+  name: 'Company',
```

Comment:
This is coming from translations.json which is `companyName` won't this break?

Reply:
oops, good catch